pkg/pluginmanager/manager.go

```diff
@@ -437,6 +437,7 @@ func DiscoveredFromPlugininfo(p *cli.PluginInfo) discovery.Discovered {
 		RecommendedVersion: p.Version,
 		Source:             p.Discovery,
 		SupportedVersions:  []string{p.Version},
+		Target:             p.Target,
```

Comment:
The target was not being copied from one struct to the other, which caused some unexpected failures in the new unit tests.